### DISCUSSION TITLE
Beatmap Storyboards in the Main Menu

### DISCUSSION THREAD

**Discussion by LeNitrous** [ppy/osu#13299] [Ideas] [answered]
I'll hold off opening a PR for now but I do have a working proof of concept which is shown as a video below. Basically as a supporter feature, show the beatmap's storyboard. The toggle for this feature uses the same as for the player. The settings item  under **Detail Settings** labelled "Storyboard / Video".

https://user-images.githubusercontent.com/20495991/120431973-cfea6100-c3ab-11eb-8aab-f47e6610c508.mp4

https://user-images.githubusercontent.com/20495991/120434601-43da3880-c3af-11eb-8c53-8259f4554628.mp4

**Comment by peppy** [comment]
Is this listed as a separate choice for what you see at the main menu, in addition to beatmap backgrounds? If it's to be implemented, it definitely needs to be, because I'm sure many users will want a different setting to their beatmap video/storyboard disabled setting.

If so, I think it's fine to PR it. Depends how simple and isolated the implementation is as to whether it will be accepted or not, of course.

**Reply by LeNitrous** [reply]
I can make it so if that seems to be the expected consensus.

**Reply by LeNitrous** [reply]
I was thinking of something like this because I think this makes sense. Some beatmaps don't have videos or storyboards which then fallback to the static beatmap background.
![image](https://user-images.githubusercontent.com/20495991/120441729-72f4a800-c3b7-11eb-84f6-cac28db45899.png)

**Reply by peppy** [reply]
It should be in the dropdown. Not yet another setting. The setting doesn't even make sense unless you have one specific item in the dropdown selected in the first place.

> Beatmap
> Beatmap (with storyboard / video)

or something like that.

**Comment by frenzibyte** [comment, marked as answer]
Has been supported now (https://github.com/ppy/osu/pull/13303).
